docs/integrate/cdc/index.md

```diff
@@ -17,6 +17,32 @@ to use them optimally.
 Please also have a look at support for [generic ETL](#etl) solutions.
 :::
 
+
+## Amazon DynamoDB
+:::{div}
+Tap into [Amazon DynamoDB Streams], to replicate CDC events from DynamoDB into CrateDB,
+with support for CrateDB's container data types.
+- {hyper-open}`Documentation <[DynamoDB CDC Relay]>`
+- {hyper-read-more}`Blog <[Replicating CDC events from DynamoDB to CrateDB]>`
+:::
+
+## AWS DMS
+[AWS Database Migration Service] (AWS DMS) is a managed migration and replication
+service that helps move your database and analytics workloads quickly, securely,
+and with minimal downtime and zero data loss.
+
+AWS DMS supports migration between 20-plus database and analytics engines, either
+on-premises, or per EC2 instance databases. Supported data migration sources are:
+Amazon Aurora, Amazon DocumentDB, Amazon S3, IBM DB2, MariaDB, Azure SQL Database,
+Microsoft SQL Server, MongoDB, MySQL, Oracle, PostgreSQL, SAP ASE.
+
+CrateDB provides two variants how to conduct data migrations using AWS DMS.
+Either use it standalone / on your own premises, or use it in a completely
+managed environment with services of AWS and CrateDB Cloud.
+
+- {hyper-navigate}`AWS DMS Kinesis Relay (standalone) <[AWS DMS Kinesis Relay]>`
+- {hyper-navigate}`AWS DMS Kinesis Relay (managed) <[AWS DMS Kinesis Relay]>`
```

Comment:
Links targets are a work in progress. See:

- https://github.com/crate/cratedb-toolkit/pull/213